examples/general/config-array/webpack.config.js

```diff
@@ -47,11 +54,16 @@ module.exports = [
           test: /\.png$/,
           loader: 'url-loader',
           options: { limit: 100000 }
+        },
+        {
+          test: /\.(woff|woff2|eot|ttf|otf)$/,
+          use: 'url-loader?limit=10000&name=font/[hash].[ext]'
+        },
+        {
+          test: /\.svg$/,
+          use: 'url-loader?limit=8192&name=image/[hash].[ext]'
         }
       ]
-    },
```

Comment:
Please enable `minimize` for this example

Reply:
Sorry, my fault, please check again. I also remove bundle.js from layout.